examples/gallery/plot/inset-rectangle.py

```diff
@@ -0,0 +1,48 @@
+"""
+Inset with an rectangle
+------------------------
+
+The :meth:`pygmt.Figure.inset` method adds an inset figure inside a larger
+figure. The function is called using a ``with`` statement, and its position,
+box, offset, and margin parameters are set. Plotting methods called within the
+``with`` statement plot into the inset figure.
+"""
+
+import numpy as np
+import pygmt
+
+# Set the region to be near Tokyo
+region = [139.2, 140.5, 34.8, 36]
```

Comment:
Could you try a large region? This region is too small, making the blue rectangle difficult to see in the inset map.

Reply:
Done in 5748ef7.